src/Covid19Dashboard/dataUtils.js

```diff
@@ -23,3 +23,35 @@ export async function formatSeirData(tsvData) {
   });
   return contents;
 }
+
+export async function formatTop10Data(tsvData) {
+  let headers = null;
+  let dateIndex;
+  const contents = {};
+  papaparse.parse(tsvData, {
+    worker: true,
+    skipEmptyLines: true,
```

Comment:
could you please replace `if (row.data.length >= 2) { // skip empty line at end of file` in `formatSeirData` with `skipEmptyLines: true,` as well? 😬

Reply:
Done. 👍